requirements.txt

```diff
@@ -1,6 +1,6 @@
 -e git+https://github.com/GSA/ckanext-datagovtheme.git#egg=ckanext-datagovtheme
 -e git+https://github.com/GSA/ckanext-datajson.git#egg=ckanext-datajson
--e git+https://github.com/ckan/ckanext-harvest.git#egg=ckanext-harvest
+-e git+https://github.com/ckan/ckanext-harvest.git@89b1a322acfcdabdf3274d3c47dd95291c9c5427#egg=ckanext_harvest
```

Comment:
Is there something wrong with the latest ckanext-harvest? If so, do we know what?